### ISSUE TITLE
Palantir decision events

### ISSUE THREAD

**Issue by BriceLeduc** [MattTheLegoman/RealmsInExile#405]
"Gaze into the Palantír" decision is now tied to possesion of one.
We need new events, especially covering the areas introduced in 2.2 and 3.0.

The decisions will be split according to the alignment of the character :

- the good and neutral will have access to distant visions, but twisted by the influence of Sauron. Basically you've spent too much time on Facebook conspiracy theories groups and think everyone is out to get you

- the evil version should be more like any authoritarian state TV: everything at home is fine thanks to the wise overlord Sauron, but the decadent forces of "Good" are ever a threat and need to be neutralised if the world wants peace. Sauron can also contact you to test you or request something (a prisoner, a tribute ?)

- Sauron and the Nazgûl might need custom ones, as they are not being fooled by the Dark Tower

I can write the event file myself, or at least tweak things once you're done

**Comment by BriceLeduc** [comment]
Activity successfully renovated

**Comment by BriceLeduc** [comment]
Things to be tested :

- cost/benefit balance : is the chance to get a lifestyle perk point high enough to warrant the cost of the event ?
- does the Sauron event happen often enough/too often ?
- is the madness chance of the Sauron event set too high/too low ?

Console command : **event palantir.1**

**Comment by blackbird5598** [comment]
> Things to be tested :
> 
> * [ ]  cost/benefit balance : is the chance to get a lifestyle perk point high enough to warrant the cost of the event ?
> * [ ]  does the Sauron event happen often enough/too often ?
> * [ ]  is the madness chance of the Sauron event set too high/too low ?
> 
> Console command : **event palantir.1**

Whenever I try it it usually ends after the first place you see through the palantir, I almost never have the option to see the next one or put the palantir down and if I do it usually ends on that one. (Saruman, Denethor, Aragorn) have not tried Sauron yet.

**Comment by savagesach** [comment]
I agree, I've had the same experiences. I would also add that the cost for saruman is nonexistent so there's no reason not to click the button in his case.

**Comment by BriceLeduc** [comment]
> > Things to be tested :
> > 
> > * [ ]  cost/benefit balance : is the chance to get a lifestyle perk point high enough to warrant the cost of the event ?
> > * [ ]  does the Sauron event happen often enough/too often ?
> > * [ ]  is the madness chance of the Sauron event set too high/too low ?
> > 
> > Console command : **event palantir.1**
> 
> Whenever I try it it usually ends after the first place you see through the palantir, I almost never have the option to see the next one or put the palantir down and if I do it usually ends on that one. (Saruman, Denethor, Aragorn) have not tried Sauron yet.

I don't understand. What option do you get then ? Those are the only two I coded

**Comment by BriceLeduc** [comment]
> I agree, I've had the same experiences. I would also add that the cost for saruman is nonexistent so there's no reason not to click the button in his case.

Yes, the whole activity design pre-dates the decision to reduce stress for immortals. I don't know how else to do it, though
